mcp_server_snowflake/server.py

```diff
@@ -240,6 +240,16 @@ def _get_persistent_connection(
                 logger.info("Using external authentication")
                 connection_params = self.connection_params.copy()
 
+            # We are passing session_parameters and client_session_keep_alive
+            # so we cannot rely on the connection to infer default connection name.
+            # So instead, if no expliit values passed via CLI, we replicate the same logic here
```

Comment:
```suggestion
            # So instead, if no explicit values passed via CLI, we replicate the same logic here
```

Hopefully today's smallest nit from me haha 😓